fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyTranslator.java

```diff
@@ -179,11 +179,24 @@ public String dualTableName() {
     @Override
     public String selectSequenceNextValue(String schemaName, String sequenceName) {
         final String qname = DataDefinitionUtil.getQualifiedName(schemaName, sequenceName);
-        return "SELECT NEXT VALUE FOR " + qname + " FROM SYSIBM.SYSDUMMY1";
+        // return "SELECT NEXT VALUE FOR " + qname + " FROM SYSIBM.SYSDUMMY1";
```

Comment:
interesting code from the prior commit. 
Let's remove this commented line

Reply:
yeah, probably cruft left over from my search for the Derby/Liberty alter table bug.